vscode/webviews/components/UserMenu.tsx

```diff
@@ -262,6 +266,91 @@ export const UserMenu: React.FunctionComponent<UserMenuProps> = ({
+                                        })
+                                        close()
+                                    }}
+                                >
+                                    <span className="tw-flex-grow">Enable Debug Mode</span>
+                                </CommandItem>
+
+                                <CommandItem
+                                    onSelect={() => {
+                                        getVSCodeAPI().postMessage({
+                                            command: 'command',
+                                            id: 'cody.debug.outputChannel',
+                                        })
+                                        close()
+                                    }}
+                                >
+                                    <span className="tw-flex-grow">Open Output Channel</span>
+                                </CommandItem>
+                            </CommandGroup>
+                        </CommandList>
+                    ) : userMenuView === 'help' ? (
+                        <CommandList>
+                            <CommandGroup title="Help Menu">
+                                <CommandItem onSelect={() => onMenuViewChange('main')}>
+                                    <ChevronLeftIcon size={16} strokeWidth={1.25} className="tw-mr-2" />
+                                    <span className="tw-flex-grow">Back</span>
+                                </CommandItem>
+                            </CommandGroup>
+                            <CommandGroup>
+                                <CommandLink
+                                    href="https://community.sourcegraph.com/"
+                                    target="_blank"
+                                    rel="noreferrer"
+                                    onSelect={() => {
+                                        telemetryRecorder.recordEvent(
+                                            'cody.userMenu.helpLink',
+                                            'open',
+                                            {}
+                                        )
+                                        close()
+                                    }}
+                                >
+                                    <span className="tw-flex-grow">Sourcegraph Community</span>
+                                    <ExternalLinkIcon size={16} strokeWidth={1.25} />
+                                </CommandLink>
+
+                                <CommandItem
+                                    onSelect={() => {
+                                        getVSCodeAPI().postMessage({
+                                            command: 'command',
+                                            id: 'cody.debug.reportIssue',
+                                        })
+                                        close()
+                                    }}
+                                >
```

Comment:
```suggestion
                                {IDE === CodyIDE.VSCode && (<CommandItem
                                    onSelect={() => {
                                        getVSCodeAPI().postMessage({
                                            command: 'command',
                                            id: 'cody.debug.reportIssue',
                                        })
                                        close()
                                    }}
                                >)}
```
I think this should be VS Code only as well.